src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -160,6 +169,17 @@ before_credit( void * _ctx,
   }
 }
 
+static void
+during_housekeeping( void * _ctx ) {
+  fd_net_ctx_t * ctx = (fd_net_ctx_t *)_ctx;
+  long now = fd_log_wallclock();
+  if( now > ctx->ip_next_upd ) {
```

Comment:
nit but kbowers style here is like

```suggestion
  if( FD_UNLIKELY( now>ctx->ip_next_upd ) ) {
```

Reply:
This isn't a style thing. If it's actually unpredictable it's better not to annotate. In this case FD_UNLIKELY is correct

Reply:
According to Kevin all branches should be annotated, the annotation determines code layout in the binary, it does not affect the branch predictor in the CPU, which can't be hinted.

Reply:
I'll take it up with KB